### PR TITLE
Release Kaptain 2.2.0-beta.1

### PR DESCRIPTION
**Pull request by alembiewski**
### What changes were proposed in this pull request?
Resolves [D2IQ-93661](https://d2iq.atlassian.net/browse/D2IQ-93661).

### Why are the changes needed?
To release Kaptain 2.2.0-beta.1

### How were the changes tested?
- N/A

### CODE REVIEW

File: services/kaptain/2.2.0/defaults/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - cm.yaml
```
Comment (MessKon) [inline services/kaptain/2.2.0/defaults/kustomization.yaml]: minor

File: services/kaptain/2.2.0/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - kaptain.yaml
+  - ../../../helm-repositories/kaptain
```
Comment (MessKon) [inline services/kaptain/2.2.0/kustomization.yaml]: minor

File: helm-repositories/kaptain/kaptain-repo.yaml [hunk 3]
```diff
@@ -7,6 +7,6 @@ metadata:
   labels:
     kommander.d2iq.io/dkp-airgapped: supported
 spec:
-  url: "${helmMirrorURL:=https://mesosphere.github.io/kaptain-charts}"
+  url: "${helmMirrorURL:=https://mesosphere.github.io/kaptain/charts}"
```
Comment (MessKon) [inline helm-repositories/kaptain/kaptain-repo.yaml:10]: will this point to the `gh-pages` branch?

Comment (alembiewski) [reply helm-repositories/kaptain/kaptain-repo.yaml:10]: Yes, GHP uses it for deployment

**Review by MessKon — APPROVED** [review]
LGTM ✅

**Pull request merged**